api/apps/geoprocessing/src/migrations/geoprocessing/1626682903111-OutputScenariosPuDataAggregate.ts

```diff
@@ -0,0 +1,38 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class OutputScenariosPuDataAggregate1626682903111
+  implements MigrationInterface {
+  name = 'OutputScenariosPuDataAggregate1626682903111';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP INDEX "output_scenarios_pu_data_idx"`);
+    await queryRunner.query(
+      `ALTER TABLE "output_scenarios_pu_data" DROP COLUMN "run_id"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "output_scenarios_pu_data" ADD "included_count" bigint NOT NULL`,
```

Comment:
this won't work on already populated databases (and making the migration fully transform from the current to the new data shape may be overkill, especially at this stage) - let's remember to take this into account when deploying on test instances where restarting from scratch may involve some backup/restore

Reply:
but if doing so, this part of the migration needs to go after the updates to the `value` column which are currently below, or PostgreSQL will run into a type mismatch in the aggregate function.

Reply:
@hotzevzl I agree.

Just thought that, as this is the first code to fill the data, there shouldn't be many issues but you are right about the seeds - will fix.

regarding using `generated` - as we are already going through values, calculating it in code does not seem to have a great load and we keep the logic in single place (API) instead separating logic between pieces. Not sure how fast calculations would be in DB when inserting per-PU and calculating it on the fly (I believe not noticeable much tho)

Reply:
@hotzevzl hm actually this drops the data (as you mentioned, migration would be an overkill, and the real inserts weren't there until this PR), so calculations wouldn't be possible (i.e. would always yield 0). Would `default 0` be fine for this very moment?

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/344/commits/0c217c0d4132235a24f954bd34d06638ac2e2b7b

Please let me know otherwise

Reply:
I think it's ok as default for any data which may be in the db by the time the migration runs, but in general I would drop the default straight away after the column has been created and populated for any existing data, so that we keep the stricter constraint on a running system.

Reply:
done https://github.com/Vizzuality/marxan-cloud/pull/344/commits/eb5566a378b022a2bf4bd114e8a9313dea0bc75b